prowler/providers/aws/services/cloudfront/cloudfront_distributions_multiple_origin_failover_configured/cloudfront_distributions_multiple_origin_failover_configured.metadata.json

```diff
@@ -0,0 +1,34 @@
+{
+  "Provider": "aws",
+  "CheckID": "cloudfront_distributions_multiple_origin_failover_configured",
+  "CheckTitle": "Check if CloudFront distributions have origin failover enabled.",
+  "CheckType": [
+    "NIST 800-53 Controls"
```

Comment:
Please use the types from https://docs.aws.amazon.com/securityhub/latest/userguide/asff-required-attributes.html#Types